spec/cucumber/formatter/legacy_api/adapter_spec.rb

```diff
@@ -1703,7 +1703,7 @@ def apply_before_hooks(test_case)
             end
 
             expect( formatter.legacy_messages ).to eq([
-                :before_features,
+                                                        :before_features,
                   :before_feature,
```

Comment:
I think we could safely exclude this file from linting, I hope we'll be able to remove it from the project altogether fairly soon.